src/batch-client/index.ts

```diff
@@ -1088,12 +1094,20 @@ export class ZimbraBatchClient {
 	/**
 	 * These options are included on every request.
 	 */
-	private getAdditionalRequestOptions = () => ({
-		jwtToken: this.jwtToken,
-		sessionId: this.sessionId,
-		origin: this.origin,
-		userAgent: this.userAgent
-	});
+	private getAdditionalRequestOptions = () => {
+		if (this.sessionId === '1') {
```

Comment:
instead of checking on default value, we can always read from cache for first time and define default value there
subsequent calls will read this.sessionId

Reply:
updated